docs/alphafeatures.md

```diff
@@ -12,1239 +12,64 @@ This section contains different features of OpenEBS which are presently in Alpha
 **Note** : Upgrade is not supported for features in Alpha version.
 
 
-## cStor
+## Mayastor
 
-[Running a sample application on a cStor volume provisioned via CSI provisioner](#running-sample-application-cstor-volume-using-csi-provisioner)
+OpenEBS Mayastor developed using NVMe based architecture, targeted at addressing performance requirements of IO-intensive workloads is ready for alpha testing. For detailed instructions on how to get started with Mayastor please refer to this [Quick-start guide](https://github.com/openebs/Mayastor/blob/master/doc/quick.md).
 
-[Provisioning cStor pool using CSPC operator](#provision-cstor-pool-using-cspc-operator)
+The following features are supported:
+- Creation of a Mayastor pool using block device attached to the node.
```

Comment:
block device -> block devices ?

Rest of the changes are good..

Reply:
currently, it only supports a single block device.